### DISCUSSION TITLE
Did not record for a zoom call

### DISCUSSION THREAD

**Discussion by lerenah** [ExistentialAudio/BlackHole#714] [General]
### Please Confirm

- [X] I have read the **[FAQ](https://github.com/ExistentialAudio/BlackHole#faq) and [Wiki](https://github.com/ExistentialAudio/BlackHole/wiki)** where most common issues can be resolved
- [X] I have searched **[Discussions](https://github.com/ExistentialAudio/BlackHole/discussions)** to see if the same question has already been asked
- [X] This is a **bug** and not a question about audio routing or configuration, which should be posted in [Discussions](https://github.com/ExistentialAudio/BlackHole/discussions)

### macOS Version

macOS 11 Big Sur

### BlackHole Build(s) Affected

- [X] 2 channel
- [ ] 16 channel
- [ ] 64 channel
- [ ] other/custom build

### Describe the bug

I tested this using a Youtube video and it worked.  But when I went to record a very important zoom call, I was surprised to see that I had no audio for that call at all.  Is this the expected behavior and if so, is there a disclaimer I should be aware of?

### Reproduction Steps

1. Setup audio Midi after downloading BlackHole 2ch
2. Created Aggregate Device for Screen input with/Audio
3. Checked Speakers and Blackhole option
4. Selected QuickTIme Player Input and checked BlackHold 2ch option
5. Recorded on QuickTime, selected Options and selectedQuickTime Player

Followed steps from this youtube channel: https://www.youtube.com/watch?v=prUVS0HF2gU&t=272s

### Expected Behavior

Expected to have sound in the recording of a zoom call.

**Comment by devinroth** [comment]
I just did a quick test to see if anything has changed. And I had absolutely no issues recording Zoom. I suspect you had it setup wrong. 

As for a disclaimer, BlackHole is an open source project under the GPL3.0 License. You can find find the disclaimer at section 15. https://github.com/ExistentialAudio/BlackHole/blob/master/LICENSE. 

In the future, I highly suggest you do a test recording with Zoom to make sure you have it properly setup before an important call. 

If you need help, post some screenshots of your setup, audio midi setup, Zoom, QT etc.

**Reply by lerenah** [reply]
Thanks for clarifying.  The confusion comes in because I did a test run before the call, but using YouTube and was able to get sound.  So are the settings different when recording a Zoom call?

**Comment by devinroth** [comment]
Zoom uses different audio settings than your web browser.

**Reply by lerenah** [reply]
Ok are there instructions on how to use Blackhole to record a zoom call?

**Comment by devinroth** [comment]
Should be the same except you need to set the output in Zoom to your Multi-output.

**Comment by lerenah** [comment]
Ok thanks
